### ISSUE TITLE
Freeradius fails to start with DHCP enabled

### ISSUE THREAD

**Issue by benedikt-bartscher** [opnsense/plugins#3261]
**Describe the bug**
Freeradius fails to start with DHCP enabled

**To Reproduce**
1. Enable Radius with SQLLite and ensure it works
2. Add a Radius DHCP Server
3. Enable Raidus DHCP Server Support

**Expected behavior**
Radius starts with DHCP Server

**Screenshots**
![image](https://user-images.githubusercontent.com/31854409/212545340-4cfb1442-5a6b-46ca-80a0-b2e85d20cd78.png)
![image](https://user-images.githubusercontent.com/31854409/212545370-fd30c360-8ae4-4451-b644-c3cc40d117a0.png)

**Relevant log files**
```
root@fw:~ # radiusd -X
FreeRADIUS Version 3.2.1
Copyright (C) 1999-2022 The FreeRADIUS server project and contributors
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE
You may redistribute copies of FreeRADIUS under the terms of the
GNU General Public License
For more information about these matters, see the file named COPYRIGHT
Starting - reading configuration files ...
including dictionary file /usr/local/share/freeradius/dictionary
including dictionary file /usr/local/share/freeradius/dictionary.dhcp
including dictionary file /usr/local/share/freeradius/dictionary.vqp
including dictionary file /usr/local/etc/raddb/dictionary
including configuration file /usr/local/etc/raddb/radiusd.conf
including configuration file /usr/local/etc/raddb/proxy.conf
including configuration file /usr/local/etc/raddb/clients.conf
including files in directory /usr/local/etc/raddb/mods-enabled/
including configuration file /usr/local/etc/raddb/mods-enabled/attr_filter
including configuration file /usr/local/etc/raddb/mods-enabled/chap
including configuration file /usr/local/etc/raddb/mods-enabled/date
including configuration file /usr/local/etc/raddb/mods-enabled/detail
including configuration file /usr/local/etc/raddb/mods-enabled/detail.log
including configuration file /usr/local/etc/raddb/mods-enabled/digest
including configuration file /usr/local/etc/raddb/mods-enabled/dynamic_clients
including configuration file /usr/local/etc/raddb/mods-enabled/eap
including configuration file /usr/local/etc/raddb/mods-enabled/echo
including configuration file /usr/local/etc/raddb/mods-enabled/exec
including configuration file /usr/local/etc/raddb/mods-enabled/expiration
including configuration file /usr/local/etc/raddb/mods-enabled/expr
including configuration file /usr/local/etc/raddb/mods-enabled/files
including configuration file /usr/local/etc/raddb/mods-enabled/linelog
including configuration file /usr/local/etc/raddb/mods-enabled/logintime
including configuration file /usr/local/etc/raddb/mods-enabled/mschap
including configuration file /usr/local/etc/raddb/mods-enabled/ntlm_auth
including configuration file /usr/local/etc/raddb/mods-enabled/pap
including configuration file /usr/local/etc/raddb/mods-enabled/passwd
including configuration file /usr/local/etc/raddb/mods-enabled/preprocess
including configuration file /usr/local/etc/raddb/mods-enabled/radutmp
including configuration file /usr/local/etc/raddb/mods-enabled/realm
including configuration file /usr/local/etc/raddb/mods-enabled/replicate
including configuration file /usr/local/etc/raddb/mods-enabled/soh
including configuration file /usr/local/etc/raddb/mods-enabled/sradutmp
including configuration file /usr/local/etc/raddb/mods-enabled/totp
including configuration file /usr/local/etc/raddb/mods-enabled/unix
including configuration file /usr/local/etc/raddb/mods-enabled/unpack
including configuration file /usr/local/etc/raddb/mods-enabled/sql
including configuration file /usr/local/etc/raddb/mods-config/sql/main/sqlite/queries.conf
including configuration file /usr/local/etc/raddb/mods-enabled/utf8
including configuration file /usr/local/etc/raddb/mods-enabled/always
including configuration file /usr/local/etc/raddb/mods-enabled/counter
including configuration file /usr/local/etc/raddb/mods-enabled/dhcp_sqlippool
including configuration file /usr/local/etc/raddb/mods-config/sql/ippool-dhcp/mysql/queries.conf
/usr/local/etc/raddb/mods-config/sql/ippool-dhcp/mysql/queries.conf[34]: Reference "${req_attribute_name}" not found
/usr/local/etc/raddb/mods-config/sql/ippool-dhcp/mysql/queries.conf[127]: Reference "${offer_duration}" not found
including configuration file /usr/local/etc/raddb/mods-enabled/ldap
including configuration file /usr/local/etc/raddb/mods-enabled/mac2ip
including configuration file /usr/local/etc/raddb/mods-enabled/sqlippool
including configuration file /usr/local/etc/raddb/mods-config/sql/ippool/sqlite/queries.conf
including files in directory /usr/local/etc/raddb/policy.d/
including configuration file /usr/local/etc/raddb/policy.d/accounting
including configuration file /usr/local/etc/raddb/policy.d/canonicalization
including configuration file /usr/local/etc/raddb/policy.d/control
including configuration file /usr/local/etc/raddb/policy.d/cui
including configuration file /usr/local/etc/raddb/policy.d/debug
including configuration file /usr/local/etc/raddb/policy.d/dhcp
including configuration file /usr/local/etc/raddb/policy.d/eap
including configuration file /usr/local/etc/raddb/policy.d/filter
including configuration file /usr/local/etc/raddb/policy.d/moonshot-targeted-ids
including configuration file /usr/local/etc/raddb/policy.d/operator-name
including configuration file /usr/local/etc/raddb/policy.d/rfc7542
including configuration file /usr/local/etc/raddb/policy.d/abfab-tr
including files in directory /usr/local/etc/raddb/sites-enabled/
including configuration file /usr/local/etc/raddb/sites-enabled/inner-tunnel
including configuration file /usr/local/etc/raddb/sites-enabled/default
including configuration file /usr/local/etc/raddb/sites-enabled/dhcp
including configuration file /usr/local/etc/raddb/sites-enabled/check-eap-tls
/usr/local/etc/raddb/mods-config/sql/ippool-dhcp/mysql/queries.conf[34]: Reference "${req_attribute_name}" not found
Errors reading or parsing /usr/local/etc/raddb/radiusd.conf
```

**Environment**
OPNsense 22.7.10_2-amd64
FreeBSD 13.1-RELEASE-p5
OpenSSL 1.1.1s 1 Nov 2022

**Comment by mimugmail** [comment]
What happens without sqlite? Not at a computer right now

**Comment by Dustie** [comment]
Without **Services -> FreeRADIUS -> General -> Enable SQLite** ?

That is disabled.

**Comment by mimugmail** [comment]
Freeradius maintainers are config breakers :) Need to take a closer look

**Comment by mimugmail** [comment]
For demo purposes, yes. But you need to disable kea and ISC to get it running

**Comment by yelabuga** [comment]
I am sorry for confusing. It seems I didnt understand properly.

In real, I would like to ask - can my OPNSense router act as a wifi Access Point? Out of the box I mean.
OPNSense on my router "sees" my wifi adapter/interface 'ath', and I have created WLAN interface on it.

I even needn't DHCP (I could set the IP address and gw directly on the phone) and I even neednt Raduis auth.

Sorry for confusing, probably I chose wrong place to ask (facepalm on me)

thank you.
